k8s/src/main/scala/io/buoyant/k8s/istio/MixerClient.scala

```diff
@@ -25,14 +26,17 @@ private[telemetry] object MixerClient {
     mapper.readValue[Seq[String]](yaml).asInstanceOf[Seq[String]]
   }
 
-  private[telemetry] def indexOf(dict: Seq[String], str: String): Int =
+  private[this] def indexOf(dict: Seq[String], str: String): Int =
     -1 * (dict.indexOf(str) + 1)
 
-  private[telemetry] def mkReportRequest(): ReportRequest = {
+  // TODO: http1-only for now
+  private[istio] def mkReportRequest(req: Request, rsp: Option[Response]): ReportRequest = {
```

Comment:
if you make this take a `Try[Response]` instead of an `Option[Response]` then we'll have access to the exception in the case of a failure.